src/main/java/api/apiwrappers/PetsApiWrapper.java

```diff
@@ -0,0 +1,61 @@
+package api.apiwrappers;
+
+import java.io.IOException;
+import api.CommonApiWrapper;
+import api.domain.pets.*;
+
+public class PetsApiWrapper extends CommonApiWrapper{
+    public Pet postPet(Pet pet) throws IOException {
+        String endpoint = baseUrl + "/v2/pet";
```

Comment:
You can add `baseUrl + "/v2/pet"` in a global variable, since it's used inside all the methods:
`private String petEndpoint = baseUrl + "/v2/pet";`

Reply:
I've changed `PetApiWrapper` to reflect the suggestion.